src/popup/baseItemList.js

```diff
@@ -242,6 +242,17 @@ export default class BaseItemList extends HTMLElement {
     let searchInput = this.search;
     let listBody = this.shadowRoot.querySelector(".list-body");
 
+    const observer = new MutationObserver(() => {
+      if (!this.selected) {
+        this.selected = this.nthItem(1);
+      }
+    });
+
+    observer.observe(listBody, {
+      childList: true,
+      subtree: true
+    });
```

Comment:
I suspect this mutation observer will fire a lot when typing, or in general. Is it possible to move this logic to _addItem or something similar, where the selected item is simply set after additions are complete?

Reply:
I moved it in the constructor and it works. It was really an error adding the observer in that method. It should be added only once.
I updated the PR.